WMFComponents/Sources/WMFComponents/Components/Article Tabs/WMFArticleTabsView.swift

```diff
@@ -105,35 +116,36 @@ public struct WMFArticleTabsView: View {
                                 image
                                     .resizable()
                                     .scaledToFill()
-                                    .frame(width: geo.size.width, height: 95)
+                                    .frame(width: geo.size.width, height: CGFloat(viewModel.calculateImageHeight()))
                                     .clipped()
                             } placeholder: {
                                 Color(uiColor: theme.paperBackground)
-                                    .frame(width: geo.size.width, height: 95)
+                                    .frame(width: geo.size.width, height: CGFloat(viewModel.calculateImageHeight()))
                             }
                         }
                     } else {
                         tabTitle(title: tab.title)
                             .frame(maxWidth: .infinity, alignment: .leading)
                             .padding(.trailing, 40)
-                            .padding(.top, 12)
                             .padding(.leading, 10)
                     }
                 }
 
                 if viewModel.shouldShowCloseButton {
-                    WMFCloseButton(action:{
+                    WMFCloseButton(action: {
```

Comment:
I notice that when double-tapping while the tab is highlighted seems to close the tab. I think when activating the tab, it should allow you to visit that tab. (See video).


https://github.com/user-attachments/assets/ac54d28e-df74-45e7-86cd-e31a8884e670

Reply:
I think your most recent commit may have changed the behavior. Now it goes to the article upon double-tap, but it doesn't seem to read out row 2 (see new video). I'm also not sure how to close the tab now, let me know if I'm missing some steps there. Thanks!

Uploading ScreenRecording_05-14-2025 16-00-14_1.MP4…

Reply:
https://github.com/user-attachments/assets/52e3e99d-52c6-4c50-bde1-36c4a16d8ad9